bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/parser/Main.java

```diff
@@ -94,9 +112,15 @@ public static void main(String[] args) throws ParseException, IOException {
     logger.info(
         "Processing finished."
             + "Queries read: {}. "
+            + "Fully parsed queries: {}. "
+            + "Partially parsed queries: {}. "
```

Comment:
What is the difference between partially parsed and fully parsed?

Reply:
This is the new feature, I think the most important one.
If all parsers were able to parse the query with no issue, the query is fully parsed. If only some parsers (at least one) is able to parse the query with no issue, the query is partially parsed. If 0 parser was able to parse the query with no issue, it is not parsed.